sdk/python/tests/compiler/testdata/compose.py

```diff
@@ -95,6 +95,6 @@ def __init__(self, name, url):
   name='Download and Save Most Frequent',
   description='Download and Get Most Frequent Word and Save to GCS'
 )
-def download_save_most_frequent_word(url: dsl.PipelineParam, outputpath: dsl.PipelineParam):
+def download_save_most_frequent_word(url: str, outputpath: str):
```

Comment:
This type annotation is not used by DSL, right? it served to show the types to the users?

Reply:
It is not used by DSL compiler.